network/builder/src/builder.rs

```diff
@@ -238,11 +238,13 @@ impl NetworkBuilder {
                 .seed_addrs
                 .iter()
                 .map(|(peer_id, addrs)| {
-                    (peer_id, Peer::from_addrs(PeerRole::Upstream, addrs.clone()))
+                    (peer_id, Peer::from_addrs(PeerRole::ValidatorFullNode, addrs.clone()))
```

Comment:
Well, technically this doesn't always have to be true, but it should be fine given the backwards compatibility (and eventual removal)